package.json

```diff
@@ -13,7 +13,9 @@
     "lint:check": "npm run lint:eslint && npm run lint:prettier -- --check",
     "lint:eslint": "eslint --config .eslintrc.js \"src/\" \"e2e/\" \"examples/\"",
     "lint:prettier": "prettier \"{src,e2e,examples}/*.{ts,tsx,js,jsx,css}\" \"**/*.{md,mdx,yml}\"",
-    "test": "jest src",
+    "test": "npm run test:unit:browser && npm run test:unit:node",
+    "test:unit:browser": "jest --testPathIgnorePatterns=e2e --testPathIgnorePatterns=node.test.ts",
+    "test:unit:node": "jest --env=node --testPathIgnorePatterns=e2e --testPathIgnorePatterns=browser.test.ts --coverage=false",
```

Comment:
These lines look round the wrong way?

Reply:
Oh, wait, you're ignoring at the end.. why not just select the file you want to test explicitly?